From 485062cfc7c933e24997a2af6118d3d289c8ec71 Mon Sep 17 00:00:00 2001
From: Andrew Wang <wang@apache.org>
Date: Tue, 2 Feb 2016 11:56:37 -0800
Subject: [PATCH 1169/2748] HADOOP-12755. Fix typo in defaultFS warning
 message.

(cherry picked from commit 8171874dd198a6d10f48211f311595b222e6b930)
(cherry picked from commit b4ad58796b70954804ad96f0c15381ad13e7a8ea)

Change-Id: I76bda954c72812cbc240ab72ab5cb1e45b600221
---
 .../java/org/apache/hadoop/fs/shell/FsCommand.java |    2 +-
 .../java/org/apache/hadoop/fs/shell/TestLs.java    |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/FsCommand.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/FsCommand.java
index b5544fc..b34224e 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/FsCommand.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/shell/FsCommand.java
@@ -111,7 +111,7 @@ protected void processRawArguments(LinkedList<String> args)
           defaultFs == null || defaultFs.equals(FS_DEFAULT_NAME_DEFAULT);
       if (missingDefaultFs) {
         err.printf(
-            "Warning: fs.defaultFs is not set when running \"%s\" command.%n",
+            "Warning: fs.defaultFS is not set when running \"%s\" command.%n",
             getCommandName());
       }
     }
diff --git a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestLs.java b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestLs.java
index b0d879a..719dd28 100644
--- a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestLs.java
+++ b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestLs.java
@@ -45,7 +45,7 @@ private static void displayWarningOnLocalFileSystem(boolean shouldDisplay)
     ls.err = err;
     ls.run("file:///.");
     assertEquals(shouldDisplay, buf.toString().contains(
-        "Warning: fs.defaultFs is not set when running \"ls\" command."));
+        "Warning: fs.defaultFS is not set when running \"ls\" command."));
   }
 
   @Test
-- 
1.7.9.5

